cfme/common/provider.py

```diff
@@ -127,11 +127,20 @@ def get_mgmt_system(self):
         """
         # gotta stash this in here to prevent circular imports
         from utils.providers import get_mgmt
+        from cfme.containers.provider.openshift import OpenshiftProvider
+
+        if isinstance(self, OpenshiftProvider):
+            credentials = conf.cfme_data['management_systems'].get(
+                self.key)['endpoints']['default']['credentials']
+            credentials = conf.credentials[credentials]
+            credentials['token'] = self.get_bearer_token()
+        else:
+            credentials = None
```

Comment:
**REQUIRED**
Please override `get_mgmt_system()` method in `OpenshiftProvider`.